infrastructure/environments/development.tfvars

```diff
@@ -126,6 +130,15 @@ avd_vm_count          = 1
 avd_users_group_name  = "DToS-hub-dev-uks-hub-virtual-desktop-User-Login"
 avd_admins_group_name = "DToS-hub-dev-uks-hub-virtual-desktop-User-ADMIN-Login"
 
+dns_zone_name                = "dev.nationalscreening.nhs.uk"
+dns_zone_resource_group_name = "rg-hub-dev-uks-public-dns-zones"
+
+lets_encrypt_certificates = {
+  wildcard         = "*.dev.nationalscreening.nhs.uk"
+  wildcard_private = "*.private.dev.nationalscreening.nhs.uk"
+}
+lets_encrypt_contact_email = "josiel.souza1@nhs.net"
```

Comment:
You really want to put a personal email in here? Should it not be a shared mailbox as Josiel might roll off the project in the future he will not have access to that mailbox.

Reply:
Moved to variable group